docs/authentication/jwt-auth.html

```diff
@@ -16,13 +16,407 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="../../css/reset.css">
     <link rel="stylesheet" href="../../css/docs.css">
+    <link rel="stylesheet" media="screen and (max-width: 900px)" href="../../css/docs_mobile.css">
     <link href=“https://fonts.googleapis.com/css?family=IBM+Plex+Sans” rel=“stylesheet”>
   </head>
   <body>
-    <h1 class="heading-1">JWT Authentication</h1>
+      <main>
+      <h1 class="heading-1">JSON Web Token Authentication</h1>
+      <p class="block-text">A JSON Web Token (JWT) defines a compact and self-contained way for securely transmitting information between parties as a JSON object. You can find out more about JWTs at `JWT.io`.</p>
```

Comment:
"parties" I think would read better as "two parties"

Reply:
`JWT.io` looks strange - why don't we make it [JWT.IO](https://jwt.io/) so it will appear as a direct link to the site, (given that the site itself has JWT.IO on its landing page I propose using the same format)?